src/app/components/experiment-workspace-detail/experiment-workspace-detail.component.html

```diff
@@ -66,23 +66,10 @@ <h3 mat-subheader>Create New Template Tab</h3>
             <qhana-template-details [templateLink]="templateLink"></qhana-template-details>
         </mat-expansion-panel>
         <mat-divider></mat-divider>
-        <h3 mat-subheader *ngIf="templateTabLinks['workspace']?.length">Workspace Tabs</h3>
-        <mat-expansion-panel *ngFor="let tab of templateTabLinks['workspace']" [expanded]="tabId == tab.resourceKey?.uiTemplateTabId"
-            (afterExpand)="selectTab(tab.resourceKey?.uiTemplateTabId ?? 'null')" (closed)="deselectTab(tab.resourceKey?.uiTemplateTabId ?? null)" 
-            [id]="'tab-' + tab.resourceKey?.uiTemplateTabId">
-            <mat-expansion-panel-header>
-                <mat-panel-title>
-                    {{tab.name}}
-                </mat-panel-title>
-                <mat-panel-description>
-                    {{templateTabObjects[tab.resourceKey?.uiTemplateTabId ?? '']?.description}}
-                </mat-panel-description>
-            </mat-expansion-panel-header>
-            <qhana-template-details [tabLink]="tab"></qhana-template-details>
-        </mat-expansion-panel>
-        <mat-divider></mat-divider>
-        <h3 mat-subheader *ngIf="templateTabLinks['experiment-navigation']?.length">Experiment Navigation Tabs</h3>
-        <mat-expansion-panel *ngFor="let tab of templateTabLinks['experiment-navigation']" [expanded]="tabId == tab.resourceKey?.uiTemplateTabId"
+
+        <ng-container *ngFor="let group of templateTabLinks | keyvalue: tabOrder">
```

Comment:
The [KeyValuePipe](https://angular.io/api/common/KeyValuePipe) is part of [`angular/core`](https://angular.io/api/core).

Since `templateTabLinks | keyvalue:tabOrder` has type `KeyValue<string, ApiLink[]>` I concatenated the `href`'s of the `ApiLink`'s:

```ts
trackByTabLink = (index: number, item: KeyValue<string, ApiLink[]>) => item.value.map(link => link.href).join(',');
```

Does that make sense?

Reply:
TrackBy should output the identity of the things that are iterated. In this case you iterate over tab groups, with the key being the group identity. Just returning `item.key` should already be fine here.